swift/kitura-nio/main.swift

```diff
@@ -0,0 +1,22 @@
+import Kitura
+
+let router = Router()
+
+router.get("/") { _, res, next in
+    try res.send("")
+    next()
+}
+
+router.get("/user/:id") {  req, res, next in
```

Comment:
can this be constrained as digit ?

Reply:
Not sure If I do understand, what exactly do you want to achieve?

Reply:
We could use codable route. For instance:
```
//contains an identifier of type `Int`
func getUserByIdHandler(id: Int, completion: (Int, RequestError?) -> Void) {
    completion(id, nil)
}

router.get("/user/:id", handler: app.getUserByIdHandler)
```

Or we could use raw request and manually convert it to Int.

Reply:
Up to you :stuck_out_tongue: 

I honestly prefer using what is considered as a `best-practice` in `swift (clean / re-usable code)

Reply:
Either are clean. Codable is model convertible language stuff. I do like raw requests style.

Reply:
Could you homogenize also for `kitura` ?

I will update result **tomorrow**